convert/lib/convert.py

```diff
@@ -223,6 +263,23 @@ def print_stack(self):
         else:
             pprint(out)
 
+    def _nested_az_replace(self, d: dict, count: int):
```

Comment:
Six of one half doesn't of the other to me, but curious why these inline functions were moved out? Since we don't have unit tests and it doesn't seem like we've changed either this or `get_stack_resources`, nor used them in any new places outside their old parent functions, I'm not sure what would inspire it.

Reply:
just moved them to make `flake8` happy
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

convert/lib/convert.py:266:5: C901 'app.generate_resource_map' is too complex (17)
convert/lib/convert.py:639:5: C901 'app.delete_stack' is too complex (19)

```